core/DataAccess/Model.php

```diff
@@ -82,7 +85,7 @@ public function getInvalidatedArchiveIdsSafeToDelete($archiveTable)
             while (!empty($duplicateArchives)) {
                 $pair = $duplicateArchives[0];
                 if (strpos($pair, '.') === false) {
-                    continue; // see below
+                    break;
```

Comment:
Maybe you could add a comment here, that this actually only happens when group_concat cut off some parts of the result. 
Also it can actually happen that `$pair` ends with an `.`. That would actually also be a cut off value, and should be handled as such. That also applies for the check below that prints the info message for it.